### PR TITLE
Meson build

### PR DESCRIPTION
**Pull request by fepitre**
QubesOS/qubes-issues#6982

### CODE REVIEW

**Comment by marmarek** [comment]
PipelineRetry

**Comment by fepitre** [comment]
> PipelineRetry

Is there something to retry here? We may add a Gitlab job indeed.

**Comment by marmarek** [comment]
I wasn't sure if the build was missing because it didn't worked, or isn't present in this repo at all. And that was the easiest method to check :)

File: gui-agent-qemu/qubes-gui.c [hunk 1]
```diff
@@ -503,6 +499,11 @@ static void qubesgui_message_handler(void *opaque)
                 case MSG_FOCUS:
                 case MSG_EXECUTE:
                     qs->vchan_data_to_discard = qs->hdr.untrusted_len;
```
Comment (jevank) [inline gui-agent-qemu/qubes-gui.c]: I guess here is the reason https://github.com/QubesOS/qubes-issues/issues/7148 - default case should be one line over.

**Pull request merged**